addons/mrp/views/mrp_production_views.xml

```diff
@@ -267,7 +267,7 @@
                                         decoration-success="not is_done and (quantity_done - should_consume_qty == 0)"
                                         decoration-warning="not is_done and (quantity_done - should_consume_qty &gt; 0.0001)"
                                         attrs="{'column_invisible': [('parent.state', '=', 'draft')], 'readonly': [('show_details_visible', '=', True)]}"/>
-                                    <field name="product_uom" attrs="{'readonly': [('state', '!=', 'draft')]}" options="{'no_open': True, 'no_create': True}" groups="uom.group_uom"/>
+                                    <field name="product_uom" attrs="{'readonly': [('state', '!=', 'draft'), ('id', '!=', False)]}" options="{'no_open': True, 'no_create': True}" groups="uom.group_uom"/>
```

Comment:
Ok, just fyi it exsits a force_save keywork that does the same but it's correct like that

Reply:
I thought of that, but then newly added lines wouldn't allow uom changes (even before saving) due to the stock.move default_get making them "Done"